CourseApp/Program.cs

```diff
@@ -6,8 +6,14 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
-            Console.ReadLine();
+            const double b = 2.5;
+            var funcalc = new FunctionCalculator();
```

Comment:
Please use `camelCase` and do not shorten the name 🔕

Reply:
Fixed